src/arch/Threads/Threads_Pthreads.cpp

```diff
@@ -208,10 +208,11 @@ MutexImpl *MakeMutex( RageMutex *pParent )
 #if defined(UNIX)
 #include <dlfcn.h>
 #include "arch/ArchHooks/ArchHooks_Unix.h"
-#elif defined(MACOSX)
-typedef int clockid_t;
-static const clockid_t CLOCK_REALTIME = 0;
-static const clockid_t CLOCK_MONOTONIC = 1;
+// commented out to allow comilation on macOS 10.12.  -dguzek
```

Comment:
I did a separate follow-up: I wanted to see if the commented lines worked for me. I also got the compilation errors you are referring to.

The purpose of this commit was to fix compilation. It's done that job. I just never encountered it for some reason. I may have to discuss how our hardware/software is different besides OSes at some point.